scan/src/pages/RequestIndexPage.re

```diff
@@ -4,8 +4,16 @@ module Styles = {
   let pageContainer = style([paddingTop(`px(20))]);
 
   let vFlex = style([display(`flex), flexDirection(`row), alignItems(`center)]);
+  let hFlex = style([display(`flex), alignItems(`center)]);
 
-  let logo = style([width(`px(50)), marginRight(`px(10))]);
+  let centerHFlex = style([display(`flex), alignItems(`center), justifyContent(`center)]);
```

Comment:
Let's make `let center = style([justifyContent(center)])` and then use `Css.merge` to combine `hFlex` and `center` together.